client/src/components/SelectTabs/EventTypesTab.js

```diff
@@ -267,7 +265,7 @@ export default function EventTypesTab() {
           <Box display="flex" className={classes.profileAndNewTypeBox}>
             <Avatar
               className={classes.avatar}
-              src={ProfileImage}
+              src=''
```

Comment:
Is this meant to be src={url} or does this work somehow?

Reply:
React Context would not pass the updated url to `EventTypesTab.js` so I removed the var until we can get a fix.